websites/V/VRV/presence.ts

```diff
@@ -1,12 +1,32 @@
 const presence = new Presence({
 		clientId: "640150336547454976",
 	}),
-	strings = presence.getStrings({
-		play: "general.playing",
-		pause: "general.paused",
-	});
+	logo = "https://i.imgur.com/p6Xv2Bv.png",
+	searchImg = "https://i.imgur.com/OIgfjTG.png",
+	playImg = "https://i.imgur.com/KNneWuF.png",
+	pauseImg = "https://i.imgur.com/BtWUfrZ.png",
+	readingImg = "https://i.imgur.com/53N4eY6.png";
```

Comment:
Put this into an array? Assets[]

Reply:
I didn't know of that thing, thanks for the advise! I'll apply the changes as soon as possible!

Reply:
Resolved in https://github.com/PreMiD/Presences/pull/6643/commits/9ea0753c14014460561121db534de68384447960